plugins/utilities.json

```diff
@@ -670,6 +670,25 @@
           "md5sum": "24913c665d05c3056c8ba390fe88155e"
         }
       }
+    },
+    "random_play": {
+      "description": "A plugin that allows you to play randomly generated FFA or Teams matches by selecting a random minigame and map for each round. This eliminates the need to set up long playlists to enjoy all your BombSquad content.",
```

Comment:
Keep the description short,we don't have much room in the plugin window for long description

Reply:
Is [527d9fc](https://github.com/bombsquad-community/plugin-manager/pull/141/commits/527d9fcd4f8864799fd2b30d9bd7d6bdd93ad6b7) better?